wiki/Medals/Unlock_requirements/Skill/de.md

```diff
@@ -0,0 +1,32 @@
+# Freischaltungsanforderungen für Skill Medaillen
+
+| Medaillenname | Anforderung |
+| :-- | :-- |
+| 500 Combo | Erziele eine 500ter Combo auf einer Beatmap in irgendeinem Spielmodus. |
+| 750 Combo | Erziele eine 750ter Combo auf einer Beatmap in irgendeinem Spielmodus. |
+| 1000 Combo | Erziele eine 1000ter Combo auf einer Beatmap in irgendeinem Spielmodus. |
+| 2000 Combo | Erziele eine 2000ter Combo auf einer Beatmap in irgendeinem Spielmodus. |
```

Comment:
```suggestion
| 500 Combo | Erziele eine 500er Combo auf einer Beatmap in irgendeinem Spielmodus. |
| 750 Combo | Erziele eine 750er Combo auf einer Beatmap in irgendeinem Spielmodus. |
| 1000 Combo | Erziele eine 1000er Combo auf einer Beatmap in irgendeinem Spielmodus. |
| 2000 Combo | Erziele eine 2000er Combo auf einer Beatmap in irgendeinem Spielmodus. |
```
Not sure but I think you just add `er` instead?

Reply:
yeah makes sense. It's `fünfhundert` `er` after all.